ac/ac-stroop/src/Dashboard.jsx

```diff
@@ -1,29 +1,35 @@
 // @flow
 
-import * as React from 'react'
-import {type LogT, dashboardViewerPropsT } from 'frog-utils'
+import * as React from 'react';
+import { type LogT, dashboardViewerPropsT } from 'frog-utils';
 
 const Viewer = ({ data }: dashboardViewerPropsT) => (
-  <pre>{JSON.stringify(data,null,2)}</pre>
-)
+  <pre>{JSON.stringify(data, null, 2)}</pre>
+);
 
 const initData = {
-  'consistent': { correct: { count: 0, time: 0 }, wrong: { count: 0, time: 0 } },
-  'inconsistent': { correct: { count: 0, time: 0 }, wrong: { count: 0, time: 0 } }
-}
+  consistent: { correct: { count: 0, time: 0 }, wrong: { count: 0, time: 0 } },
+  inconsistent: { correct: { count: 0, time: 0 }, wrong: { count: 0, time: 0 } }
+};
 
 const mergeLog = (data: any, dataFn: Object, log: LogT) => {
-  console.log(log)
-  if(log.type === 'answer' && log.payload){
-    const {isCorrect, isConsistent, answer} = log.payload
-    const stroopType = isConsistent ? 'consistent': 'inconsistent'
-    if( isCorrect === answer ){
-      dataFn.numIncr(1, [stroopType, 'correct', 'count'])
+  if (log.type === 'answer' && log.payload) {
+    const {
+      isConsistent,
+      isCorrect,
+      answer,
+      startTime,
+      answerTime
+    } = log.payload;
+    const qType = isConsistent ? 'consistent' : 'inconsistent';
+    if (isCorrect === answer) {
+      dataFn.numIncr(1, [qType, 'correct', 'count']);
+      dataFn.numIncr(answerTime - startTime, [qType, 'correct', 'time']);
     } else {
-      dataFn.numIncr(1, [stroopType, 'wrong', 'count'])
+      dataFn.numIncr(1, [qType, 'wrong', 'count']);
+      dataFn.numIncr(answerTime - startTime, [qType, 'wrong', 'time']);
```

Comment:
I don't understand this - you only store the total wrong time and total right time for all questions? Is this what the dashboard needs?

Reply:
I store the sum and the count of all the time spent on questions for each of the 4 category (inconsitent/consistent x correct/wrong). That allows to compute the averages we need for the dashboard.

Reply:
In the current dashboards, they are not distinguishing the correct/wrong but just has graphs for the inconsistent/consistent and checking to see if the student correctly marked it as wrong or not.